internal/plugin/installer/extractor.go

```diff
@@ -68,6 +68,10 @@ func NewExtractor(source string) (Extractor, error) {
 			return extractor, nil
 		}
 	}
+	if strings.HasPrefix(source, "http") && isGzipArchiveFromURL(source) {
```

Comment:
Maybe we should use the same [check](https://github.com/helm/helm/pull/31405/files#diff-3c81ef74b9a5a5d7ab90cc8dce634fb253f775552aad7055d9e2b22a6795d167R177) as in other places?